lib/Photonic/LE/NR2/SHChiTensor.pm

```diff
@@ -81,12 +81,17 @@ haydock coefficients and continued fraction.
 Returns the macroscopic second Harmonic susceptibility function for a
 given value of the dielectric functions of the host $epsA and the
 particle $epsB at the fundamental 1 and second harmonic 2 frequency.
+
 $kind is an optional letter for testing purposes with values 'd' for
-dipolar, 'q' for quadrupolar, 'e' for external and 'f' for full
-selfconsistent calculation (the default). Mask is a mask with ones and
-zeroes, to evaluate the contribution of certain regions to the
-susceptibility.
+dipolar, 'q' for quadrupolar, 'e' for external, 'l' for second harmonic
```

Comment:
This is the first time I use this method. Hope not to make many mistakes. I'd rather say:

$kind is an optional letter for testing purposes with values
'd' for dipolar contribution,
'q' for quadrupolar contribution,
'e' for external contribution (dipolar+quadrupolar),
'l' for self consistent polarization without macroscopic depolarization,
'a' for self-consistent polarization (alternative, dubious),
'el' for longitudinal projection of external polarization
'f' for full selfconsistent calculation (the default).

The result returned to the user is neither in real or reciprocal space as it is further processed, summed over, etc.